zod.go

```diff
@@ -133,12 +142,16 @@ func (c *Converter) Export() string {
 		sorted = append(sorted, ent)
 	}
 
-	sort.Sort(ByOrder(sorted))
+	sort.Sort(byOrder(sorted))
 
 	for _, ent := range sorted {
 		output.WriteString(ent.data)
 		output.WriteString("\n\n")
 	}
+
+	// Clear state
+	c.structs = 0
+	c.outputs = make(map[string]entry)
```

Comment:
Do we have to clear state?
If we use strings.Builder for example we don't clear state

```
	a := strings.Builder{}

	a.WriteString("Hello")
	a.WriteString("World")

	fmt.Println(a.String())
	fmt.Println(a.String())
```

Reply:
If we don't clear the state, calling Convert() multiple times returns schemas from all previous calls too - which IMO is not expected

Reply:
Changed to not clearing state and updated comments appropriately